examples/api/lib/material/theme/theme_extension.1..dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for ThemeExtension
+
+import 'package:flutter/material.dart';
+
+@immutable
+class MyColors implements ThemeExtension {
+  const MyColors({
+    this.blue,
+    this.red,
+  });
```

Comment:
```suggestion
  const MyColors({this.blue, this.red});
```

nit: i think this can fit in a single line. Not sure if Flutter's code style is to have constructor parameters be in multiline (-: